### PR TITLE
Install script for MariaDB

### PR DESCRIPTION
**Pull request by Landrash**
This script installs MariaDB and it's dependencies for use with the [recorder](https://home-assistant.io/components/recorder/) component in Home Assistant.
No database or database user is created during this setup and will need to be created manually after running this script.

### CODE REVIEW

**Review by ludeeus — COMMENTED** [review]
Other than the to things I commented it looks good :D

File: README.md [hunk 1]
```diff
@@ -45,6 +45,20 @@ Usually this script is not run after installation but could be used with some mo
 
 This script was originally contributed by [@Landrash](https://github.com/landrash).
 
+### Install MariaDB *(install_mariadb.sh)*
+This script installs MariaDB and it's dependencies for use with the [recorder](https://home-assistant.io/components/recorder/) component in Home Assistant.
+No database or database user is created during this setup and will need to be created manually.
+
+#### Example for creating database  
+```$ mysql -u root -p```
```
Comment (ludeeus) [inline README.md]: Had to put "sudo mysql -u root -p", or else it reported access denied.

File: README.md [hunk 2]
```diff
@@ -45,6 +45,20 @@ Usually this script is not run after installation but could be used with some mo
 
 This script was originally contributed by [@Landrash](https://github.com/landrash).
 
+### Install MariaDB *(install_mariadb.sh)*
+This script installs MariaDB and it's dependencies for use with the [recorder](https://home-assistant.io/components/recorder/) component in Home Assistant.
```
Comment (ludeeus) [inline README.md:49]: Maybe add note that this uses "MySql" connection from recorder?
Could be confusing for some.

Comment (Landrash) [reply README.md:49]: Submitting a PR to fix the documentation since it's lacking in that department.

File: README.md (hunk 1 above)
Comment (Landrash) [reply README.md]: True. Fixing.

File: README.md (hunk 2 above)
Comment (Landrash) [reply README.md:49]: Addressed in https://github.com/home-assistant/home-assistant.github.io/pull/3484 .

**Pull request merged**